rcl/src/rcl/subscription.c

```diff
@@ -213,6 +213,9 @@ rcl_subscription_get_default_options()
   if (ret == RCL_RET_OK) {
     default_options.disable_loaned_message = disable_loaned_message;
   } else {
+    RCUTILS_SAFE_FWRITE_TO_STDERR("Failed to get disable_loaned_message: ");
+    RCUTILS_SAFE_FWRITE_TO_STDERR(rcl_get_error_string().str);
+    rcl_reset_error();
```

Comment:
The same comment in regards removing logging to the error stream

Reply:
>  And by default it will be absent in most configurations.

I think that this error is important cz using `ROS_DISABLE_LOANED_MESSAGES` is meant to disable the feature. so this is strong user intention. in that case if unexpected error happens, it should be able to notify the user that `ROS_DISABLE_LOANED_MESSAGES` is not effectively working.

> This error string will appear each time we creating new pub/sub - it would be pretty annoying and could also affect performance.

I believe there should be. I think it is unlikely that this logging affects the performance, because it only happens in initialization. I think originally this PR is targeting send/receive performance improvement but initialization or finalization?

what do you think?

Reply:
@fujitatomoya Ahh never mind. 
rcl_get_disable_loaned_message(bool * disable_loaned_message) can return `RCL_RET_ERROR` only if input parameters are wrong which is not a our case any way.  
You can keep your output to the error string but error will never happened.
I was wrong on assumption that `rcl_get_disable_loaned_message(bool * disable_loaned_message)` that will return some error code if `ROS_DISABLE_LOANED_MESSAGES` env variable not defined. Sorry for confusion.

Reply:
> You can keep your output to the error stream but error will never happened.

yeah you are right, this error might print here if `something` really unexpected happens. probably never... 😅 

not at all, really appreciate your review and comments, thank you so much 👍